### DISCUSSION TITLE
commitWhenAutocommitDisabled hardwired to true in ManualSchedulerBuilder

### DISCUSSION THREAD

**Discussion by moreginger** [kagkarlsson/db-scheduler#819] [General]
Is it intentional that `commitWhenAutocommitDisabled` is hardwired to true in `ManualSchedulerBuilder`: https://github.com/kagkarlsson/db-scheduler/blob/3b79f8890e1bc55ca80bd2da1ae34c21d2ddd827/db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/testhelper/TestHelper.java#L76

I'm using `@Transactional` to rollback changes in tests, and this is causing my changes to be committed when I call `runAnyDueExecutions`.

**Comment by kagkarlsson** [comment]
I have not really thought about it, so no not really intentional. Unsure if there might be any snags if disabling it.. 
In general `ManualScheduler` probably has some improvement potential!
